data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -166,6 +168,12 @@ do
     scp -o PreferredAuthentications=publickey -o StrictHostKeyChecking=false -o UserKnownHostsFile=/dev/null -r -q "core@[${master}]:/tmp/artifacts-${GATHER_ID}/*" "${ARTIFACTS}/control-plane/${master}/"
 done
 
+cluster_bootstrap_gather
+if [ -d "${HOME}/artifacts-api-server-temp" ]
+then
+    mv "${HOME}/artifacts-api-server-temp/resources/*" "${ARTIFACTS}/resources"
+fi
```

Comment:
For this to work, the code to move from the api server artifacts from tmp to HOME has to be moved to `cluster_bootstrap_gather` (which we should do).

Do not do a wildcard move. Move the entire directory instead.
```suggestion
if [ -d "${HOME}/artifacts-api-server-temp" ]
then
    mv "${HOME}/artifacts-api-server-temp/resources" "${ARTIFACTS}"
fi
```